dev/tasks/docker-tests/github.linux.yml

```diff
@@ -41,10 +41,11 @@ jobs:
         shell: bash
         run: archery docker run -e SETUPTOOLS_SCM_PRETEND_VERSION="{{ arrow.no_rc_version }}" {{ flags|default("") }} {{ image }} {{ command|default("") }}
 
-    {% if '-r-' in image %}
+    {% if '-r' in image %}
```

Comment:
Why this change? Is this too broad?

Reply:
Before this change, some builds (e.g. `test-r-ubuntu-21.04` which uses the docker-compose service `ubuntu-r-only-r` which extends `ubuntu-r`) didn't have the log dumping step showing up at all. https://github.com/ursacomputing/crossbow/runs/3272438137?check_suite_focus=true 

The strange thing is that it _does_ have `-r-` in the underlying image name (e.g. `${REPO}:${ARCH}-ubuntu-${UBUNTU}-r-${R}`), but somehow the jinja matching isn't quite getting it there (though `-r` works). It might be overly-broad though that step _should_ be harmless since it `continue-on-error`.